src/common/localize.ts

```diff
@@ -144,3 +144,10 @@ export namespace EnvViewStrings {
     export const selectedGlobalTooltip = l10n.t('This environment is selected for non-workspace files');
     export const selectedWorkspaceTooltip = l10n.t('This environment is selected for workspace files');
 }
+
+export namespace PermissionsCommon {
+    export const allow = l10n.t('Allow');
+    export const deny = l10n.t('Deny');
+    export const ask = l10n.t('Ask');
```

Comment:
I'm not sure "Ask" is easily understandable in this context. I wonder if we should do something like "Always Ask" or "Confirm each time" at the tradeoff of having a slightly longer button title

Reply:
agreed- I also wanted this button to be explicit it was a long-term choice